contents/docs/data/sessions.mdx

```diff
@@ -138,10 +138,66 @@ This is useful for understanding entry URLs or UTMs. For example, to break down
     classes="rounded"
 />
 
+## Server SDKs and sessions
 
-## Custom session IDs
+Our server-side SDKs do not send `$session_id` by default. If you want to use sessions with server-side events, you can manually set the `$session_id` property. This lets you use session aggregations for these events and can be useful if you want to connect server-side events to session replays or web analytics sessions.
 
-Our server-side SDKs do not send `$session_id` by default. If you want to use sessions with server-side events, you can set a custom `$session_id` property. This lets you use session aggregations for these events and can be useful if you want to connect server-side events to session replays or web analytics sessions.
+You can either pass the session ID from posthog-js into your back end, or generate a new session ID. Then you can then use this session ID in your server-side events.
+
+If the events are logically part of the same session, e.g. a user starts a purchase on the front end and you receive confirmation on the back end, it's better to re-use the session ID from the front end. On the other hand, if the events have no corresponding front-end session, you can generate a new session ID.
+
+### Passing Session IDs from client-side code into server-side events
+
+Get the session ID from `posthog-js` with `posthog.get_session_id()` and send it to your server with custom code. Then, you can set the session ID on the server-side event with `posthog.capture({ ..., '$session_id': session_id })`.
+
+Here's an example of how you might do this:
+
+```javascript
+// Client side code
+
+// Get the session ID
+const posthog_session_id = posthog.get_session_id()
+
+// Make a request to your server. This is just an example, you should replace this with your own code.
+fetch('https://your-api.example.com/purchase', {
+    method: 'POST',
+    body: JSON.stringify({
+        posthog_session_id, // include the posthog session ID
+        // other data
+    }),
+    headers: {
+        'Content-Type': 'application/json'
+    }
+})
```

Comment:
I believe passing this as a `X-POSTHOG-SESSION-ID` header makes more sense. I know this is just an example, but users will simply copy-paste this and use it, so we can probably suggest better usage.

Benefits of a header:
- Much easier to parse in the backend
- No need to change their middleware logic to always accept a new `posthog_session_id` parameter
- Allows you to pass the session id for GET requests (they don't have a body)

Reply:
Makes sense, done